randovania/games/factorio/presets/starter_preset.rdvpreset

```diff
@@ -3,7 +3,7 @@
     "base_preset_uuid": null,
     "name": "Starter Preset",
     "uuid": "51f328a1-3b09-47a1-921f-b84d2cda1e4c",
-    "description": "Basic preset.",
+    "description": "This preset includes a reduced tech tree, with most repeated military upgrades removed. Recommended to play with enemies disabled.",
```

Comment:
is there a way for the mod to automatically disable enemies on export? IDK if it makes sense as a preset setting.

Reply:
Might be hackable ingame, but messing with the mapgen settings is pretty wild.

Reply:
gotcha, then wontfix it is